docs/pages/preview/upcoming-releases.mdx

```diff
@@ -10,21 +10,33 @@ The teleport team delivers a new major release roughly every 3 months.
 
 ### Teleport 9.1
 
-Teleport 9.1 will add Database and Kubernetes Access support to Machine ID and
-server-side filtering and pagination to the Web UI.
+Teleport 9.1 will add preview support for Postgres/CockroachDB backends and
+server-side filtering and pagination for CLI and the Web UI.
 
 | Version     | Date             |
 |-------------|------------------|
-| Release     | Early April 2022 |
+| Release     | April 18th 2022  |
 
 ### Teleport 9.2
 
-Teleport 9.2 will include Teleport Terminal, a native application that can be
-used to access Teleport Servers and Databases.
+Teleport 9.2 will add Database and Kubernetes Access support to Machine ID and
```

Comment:
Just to double-check, does this mean that we had clickable label filters already but we're adding Database and Kubernetes Access support to them?